src/lycophron/models.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# Lycophron is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+"""Lycophron data models."""
+
+from sqlalchemy import Column, Integer, String, JSON, create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy_utils.models import Timestamp
+
+engine = create_engine("sqlite:///" + "data.db", echo=True)
```

Comment:
shelve: we should take this from config

Reply:
I think this can be removed, no? The database is created in `db.py` using `config['SQLALCHEMY_DATABASE_URI']`